puppet/modules/profiles/manifests/web.pp

```diff
@@ -52,7 +42,6 @@
 
   class { 'web::vhost::yum':
     stable                => $stable,
```

Comment:
Perhaps align it again?
```suggestion
    stable => $stable,
```

Reply:
qouting reviewer 2: "muss das sein!?"

(ok, I'll do it)